Use of undeclared crate or module `rustix`

When compiling wasmtime 0.33 on aarch64 Linux it fails compiling with:
```rust
error[E0433]: failed to resolve: use of undeclared crate or module `rustix`
Error:   --> /cargo/registry/src/github.com-1ecc6299db9ec823/wasmtime-jit-0.33.0/src/code_memory.rs:60:13
   |
60 |             rustix::process::membarrier(
   |             ^^^^^^ use of undeclared crate or module `rustix`

error[E0433]: failed to resolve: use of undeclared crate or module `rustix`
Error:   --> /cargo/registry/src/github.com-1ecc6299db9ec823/wasmtime-jit-0.33.0/src/code_memory.rs:61:17
   |
61 |                 rustix::process::MembarrierCommand::RegisterPrivateExpeditedSyncCore,
   |                 ^^^^^^ use of undeclared crate or module `rustix`

error[E0433]: failed to resolve: use of undeclared crate or module `rustix`
Error:    --> /cargo/registry/src/github.com-1ecc6299db9ec823/wasmtime-jit-0.33.0/src/code_memory.rs:174:17
    |
174 |                 rustix::process::membarrier(
    |                 ^^^^^^ use of undeclared crate or module `rustix`

error[E0433]: failed to resolve: use of undeclared crate or module `rustix`
Error:    --> /cargo/registry/src/github.com-1ecc6299db9ec823/wasmtime-jit-0.33.0/src/code_memory.rs:175:21
    |
175 |                     rustix::process::MembarrierCommand::PrivateExpeditedSyncCore,
    |                     ^^^^^^ use of undeclared crate or module `rustix`
```
It might be related to the specific feature combination `default-features = false, features = ["cranelift", "parallel-compilation"]`

--- a/crates/jit/Cargo.toml
+++ b/crates/jit/Cargo.toml
@@ -29,10 +29,10 @@ bincode = "1.2.1"
 winapi = { version = "0.3.8", features = ["winnt", "impl-default"] }
 
 [target.'cfg(target_os = "linux")'.dependencies]
-rustix = { version = "0.31.0", optional = true }
+rustix = "0.31.0"
 
 [features]
-jitdump = ['rustix']
+jitdump = []
 vtune = ['ittapi-rs']
 
 [badges]

